cassandra/src/main/scala/filodb.cassandra/Util.scala

```diff
@@ -3,10 +3,11 @@ package filodb.cassandra
 import com.datastax.driver.core.exceptions.DriverException
 import com.datastax.driver.core._
 import java.nio.ByteBuffer
+
 import scala.concurrent.{Future, Promise}
 import scodec.bits.ByteVector
-
 import filodb.core._
+import monix.reactive.Observable
```

Comment:
Can you put the monix import in the first section with everything else?  The order should be:

everything non-filo
internal filodb